sdk/sdk-common/src/common/implementation/RiskRatio.ts

```diff
@@ -1,27 +1,46 @@
-import { type Percentage } from '~sdk-common/common/implementation/Percentage'
+import { Percentage } from '~sdk-common/common/implementation/Percentage'
 import { SerializationService } from '~sdk-common/services/SerializationService'
 
 interface IRiskRatioSerialized {
-  ratio: Percentage
+  ltv: Percentage
+}
+
+enum RiskRatioType {
+  LTV = 'LTV',
+  CollateralizationRatio = 'CollateralizationRatio',
+  Multiple = 'Multiple',
 }
 
 /**
  * @class RiskRatio
- * @description Risk ratio representing the risk of position, typically the LTV
+ * @description Risk ratio representing the risk of position, 
```

Comment:
Actually the implementation of RiskRatio in earn-sc was very good I'd like to keep that. 

In this case `typically the LTV` what does it actually mean? In fact the ltv, col ratio and multiple represents exactly the same thing and can be derived from each other.